spec.bs

```diff
@@ -2653,6 +2693,11 @@ CORP violation report=] algorithm, as leaving it unfenced may cause a privacy le
       1. Set |config|'s [=fenced frame config/embedder shared storage context=] to
          |sharedStorageContext|.
 
+      1. If |response| is non-null, set |config|'s [=fenced frame config/cross-origin reporting
```

Comment:
This is before we actually have the network response though. This `|response|` is only non-null in the x-mixed-replace and object/embed case, right? https://html.spec.whatwg.org/C#navigation-response.

I think you want this logic elsewhere.

Reply:
I've moved this to the [create and initialize a Document object](https://html.spec.whatwg.org/multipage/browsing-the-web.html#initialise-the-document-object) patches, since that's where the automatic beacon opt-in piece already lives.